.cargo/config.toml

```diff
@@ -1,8 +1,5 @@
-[target.x86_64-unknown-linux-gnu]
-rustflags = ["-C", "target-cpu=native"]
-
-[target.aarch64-apple-darwin]
+[build]
```

Comment:
Do you know if this works with the wasm build? I would be a bit afraid it tries to add the native bit in the build there too and ends up being an issue.

Reply:
I can't build wasm currently as `onig_sys v69.8.1` seams to do some funky stuff. But theoretically the more specific definition should overwrite the more generic on in a cargo config.

Reply:
Ok I just tried the `.cargo/config.toml` change locally and I can still build the wasm bit / it still works so good with me.